### ISSUE TITLE
Cannot start recaf 2.9.0

### ISSUE THREAD

**Issue by khjxiaogu** [Col-E/Recaf#290]
**Describe the bug**

Cannot start recaf because an exception was thrown.

**To Reproduce**

JDK version: 12.0.1
System: Windows 10
1. Open with cmd terminal for the first time
2. See the error

**Exception**
Full log:
```
[main] INFO : Missing JavaFX dependencies, attempting to patch in missing classes
[main] INFO :  - Local cache found!
[main] INFO :  - Loading dependencies...
[main] INFO :  - Done!
[main] INFO : Recaf-2.9.0
[main] TRACE: Loading configuration
[main] ERROR: An error occurred loading the plugins
java.lang.IllegalStateException: No 'parent' field in java.lang.ClassLoader
        at me.coley.recaf.util.VMUtil.setParent(VMUtil.java:151)
        at me.coley.recaf.plugin.PluginFolderSource.load(PluginFolderSource.java:58)
        at org.plugface.core.impl.DefaultPluginManager.loadPlugins(DefaultPluginManager.java:105)
        at me.coley.recaf.plugin.PluginsManager.load(PluginsManager.java:56)
        at me.coley.recaf.Recaf.loadPlugins(Recaf.java:108)
        at me.coley.recaf.Recaf.launch(Recaf.java:93)
        at me.coley.recaf.Recaf.main(Recaf.java:47)
Caused by: java.lang.NoSuchFieldException: parent
        at java.base/java.lang.Class.getDeclaredField(Class.java:2417)
        at me.coley.recaf.util.VMUtil.setParent(VMUtil.java:149)
        ... 6 common frames omitted
Exception in thread "WindowsNativeRunloopThread" java.lang.NoSuchMethodError: <init>
        at com.sun.glass.ui.win.WinApplication.staticScreen_getScreens(Native Method)
        at com.sun.glass.ui.Screen.initScreens(Screen.java:412)
        at com.sun.glass.ui.Application.lambda$run$1(Application.java:152)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:174)
        at java.base/java.lang.Thread.run(Thread.java:835)
Exception in thread "JavaFX Application Thread" java.lang.NullPointerException
        at com.sun.prism.d3d.D3DPipeline.getAdapterOrdinal(D3DPipeline.java:205)
        at com.sun.javafx.tk.quantum.QuantumToolkit.assignScreensAdapters(QuantumToolkit.java:738)
        at com.sun.javafx.tk.quantum.QuantumToolkit.runToolkit(QuantumToolkit.java:334)
        at com.sun.javafx.tk.quantum.QuantumToolkit.lambda$startup$10(QuantumToolkit.java:279)
        at com.sun.glass.ui.Application.lambda$run$1(Application.java:153)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:174)
        at java.base/java.lang.Thread.run(Thread.java:835)
```

**Comment by xxDark** [comment]
Will look into that, my bad.

**Comment by xxDark** [comment]
EDIT: I only fixed the first exception, the second one is not related to Recaf (?)
Does it only occur at 2.9.0 version?
See https://stackoverflow.com/questions/51101481/nosuchmethoderror-init-in-com-sun-glass-ui-win-winapplication-staticscreen-ge

**Comment by Col-E** [comment]
Github auto-closed this when merging #291 since it fixes _the first error in the log_.

As for the second, I'll defer to @xxDark on this. In the meantime, can you tell us which _vendor_ your JDK is? _(Oracle / AdoptOpenJDK / OpenJ9 / etc)_

**Comment by khjxiaogu** [comment]
> Github auto-closed this when merging #291 since it fixes _the first error in the log_.
> 
> As for the second, I'll defer to @xxDark on this. In the meantime, can you tell us which _vendor_ your JDK is? _(Oracle / AdoptOpenJDK / OpenJ9 / etc)_

Oracle

**Comment by khjxiaogu** [comment]
> EDIT: I only fixed the first exception, the second one is not related to Recaf (?)
> Does it only occur at 2.9.0 version?
> See https://stackoverflow.com/questions/51101481/nosuchmethoderror-init-in-com-sun-glass-ui-win-winapplication-staticscreen-ge
> TL;DR
> Java attempts to load wrong `glass.dll`, causing it to fail somewhere in native code.
> Remove everything from `PATH` that may point to wrong `glass.dll`
> 
> EDIT 2:
> If you don't have any `glass.dll` in PATH, then we need to go deeper.

All right, I found a jre 1.8 in my PATH, removed and works well.
